gulpfile.js

```diff
@@ -143,11 +143,12 @@ gulp.task('cloc', ['build'], function() {
         child_process.exec(cmdLine, function(error, stdout, stderr) {
             if (error) {
                 console.log(stderr);
-                return reject(error);
+                reject(error);
```

Comment:
Are these changes correct? I don't see the difference between calling `reject` and returning it